### ISSUE TITLE
Feature request: reload command

### ISSUE THREAD

**Issue by SlimeDog** [Mr-EmPee/MysticalBarriers#3]
It would be nice to have a `mb reload` command, so configuration changes could be made without having to restart the server.

**Comment by Mr-EmPee** [comment]
@SlimeDog I was thinking of making reloadable only the notifier configuration. The barriers.json isn't intended to be modified while the server is running.

First thing first, you need to know that the barrier file is saved every 2minutes. This cause several problems:
 - if you start to edit the file manually, you could not happen to have the latest configurations.
 - if you edit the file manually save it but the server trigger the periodic saving, the file that you just updated is going to be overwritten.

A solution would be saving at each barrier update the file, I think that it's not worth it. If you need to do major changes, you should stop the server or suggest a new command to update an existing barrier to do it in-game.

**Comment by SlimeDog** [comment]
Why does the JSON need to be updated every 2 minutes, if no barrier configuration has changed? It should be saved immediately on barrier changes, lest the changes be lost if the server or the plugin crashes. What am I missing?

**Comment by SlimeDog** [comment]
I disable the notifier. We have our own automagical update checker.

**Comment by Mr-EmPee** [comment]
If there are no changes, it doesn't trigger any save request. If the server crashes without triggering the disable event of the plugin, you are gonna lose the latest changes if there are any, that's a good point.

The reason why I've decided to make it periodic was because my file-manager is used on other plugins that deals with large files. At the moment, saving the barrier.json at each barrier update isn't gonna cause any harm.

I think that I will proceed with the generic reload as suggested by you and make the file save at each request.

**Comment by SlimeDog** [comment]
```
my file-manager is used on other plugins that deals with large files
```
Sure, for something like Citizens or Essentials, which saves the states of moving entities in files (that are likely to get quite large in either size or number), that makes sense. But MB states are controlled by permissions, not movement, and they are not saved in the file anyway. Thanks for reconsidering. I look forward to the update.

**Comment by Mr-EmPee** [comment]
Implemented by commit 1b6e5a7c7c7c11f3808317a78ed45fc957f47bcb
